theano/tensor/basic.py

```diff
@@ -5972,6 +6015,21 @@ def infer_shape(self, nodes, shapes):
 
 
 def diag(v, k=0):
+    """
+    A helper function for two ops: Diagonal and Diag. It both accepts tensor
```

Comment:
Again I like the concise documentation of numpy for [diag](http://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.diag.html). I would first give an overview of the behavior of the function and than explain we glue two ops together which do this and this.